# Fix FTBFS with gcc 4.4. Slightly modified patch thanks to Martin Michlmayr
--- src/display/utils.h~	2008-11-07 20:58:27.000000000 +0000
+++ src/display/utils.h	2008-11-07 20:58:35.000000000 +0000
@@ -37,6 +37,7 @@
 #ifndef RTORRENT_DISPLAY_UTILS_H
 #define RTORRENT_DISPLAY_UTILS_H
 
+#include <cstdio>
 #include <ctime>
 #include <string>
 
--- src/rpc/parse.cc~	2008-11-07 21:00:11.000000000 +0000
+++ src/rpc/parse.cc	2008-11-07 21:00:20.000000000 +0000
@@ -37,6 +37,7 @@
 #include "config.h"
 
 #include <cstring>
 #include <locale>
+#include <cstdio>
 #include <rak/path.h>
 #include <torrent/exceptions.h>
--- src/rpc/scgi_task.cc~	2008-11-07 21:01:13.000000000 +0000
+++ src/rpc/scgi_task.cc	2008-11-07 21:01:22.000000000 +0000
@@ -36,6 +36,7 @@
 
 #include "config.h"
 
+#include <cstdio>
 #include <rak/error_number.h>
 #include <sys/types.h>
 #include <sys/socket.h>
--- src/utils/lockfile.cc~	2008-11-07 21:02:36.000000000 +0000
+++ src/utils/lockfile.cc	2008-11-07 21:02:44.000000000 +0000
@@ -39,6 +39,7 @@
 #include <algorithm>
 #include <cctype>
 #include <cerrno>
+#include <cstdio>
 #include <cstring>
 #include <sstream>
 #include <fcntl.h>
--- src/command_network.cc~	2008-11-07 21:03:40.000000000 +0000
+++ src/command_network.cc	2008-11-07 21:04:54.000000000 +0000
@@ -36,6 +36,7 @@
 
 #include "config.h"
 
+#include <cstdio>
 #include <functional>
 #include <rak/address_info.h>
 #include <rak/file_stat.h>
--- src/command_ui.cc~	2008-11-07 21:04:32.000000000 +0000
+++ src/command_ui.cc	2008-11-07 21:04:41.000000000 +0000
@@ -36,6 +36,7 @@
 
 #include "config.h"
 
+#include <cstdio>
 #include <ctime>
 
 #include <rak/functional.h>

